packages/MarkdownEditor-Core.package/MarkdownBlockTextStyler.class/instance/process.into..st

```diff
@@ -0,0 +1,8 @@
+parsing
+process: aString into: aCollectionOfMarkdownBlocks
```

Comment:
I like the decomposition of the `MarkdownBlockTextStyler`'s parse method. 👍

Reply:
Thanks!